bruellware/bruellware.github.io
# made website installable as a PWA

- added theme-color meta tag
- added apple-touch-icon meta tag
- added webmanifest

To test:
- open page in chrome
- open devtools
- go to "lighthouse"
- uncheck anything but "PWA"
- run a test
> everything should be "green"

- open page in chrome
- click the "install" button in the address bar
> website should be accessible as an installed application on your system

note: firefox and safari will not allow installing PWAs from any non-https source, including localhost.
